src/Connection.ts

```diff
@@ -298,26 +320,26 @@ class Connection {
 	}
 
 	/** Get the current ISOCalendarDate from the database */
-	public async getDbDate(): Promise<string> {
-		const { timeDrift } = await this.getDbServerInfo();
+	public async getDbDate({ requestId }: GetDbDateOptions = {}): Promise<string> {
+		const { timeDrift } = await this.getDbServerInfo({ requestId });
 		return format(addMilliseconds(Date.now(), timeDrift), ISOCalendarDateFormat);
 	}
 
 	/** Get the current ISOCalendarDateTime from the database */
-	public async getDbDateTime(): Promise<string> {
-		const { timeDrift } = await this.getDbServerInfo();
+	public async getDbDateTime({ requestId }: GetDbDateTimeOptions = {}): Promise<string> {
```

Comment:
@kthompson23 The optional parameter for these functions is what is failing coverage. I'm not sure what convention would be desired to provide coverage here.

Do I create a separate test to ensure a `requestId` is generated, even though it does not pertain to this function? Do we ignore this line?

Reply:
I'd rather see us reduce the code coverage requirements here than add a pointless test or an istanbul ignore block.